python/lsst/daf/persistence/policy.py

```diff
@@ -250,6 +210,24 @@ def __contains__(self, key):
                 return False
         return keys[-1] in d
 
+    @staticmethod
+    def defaultPolicyFile(productName, filePath, repos=None):
+        """Get the path to a default policy file
+
+        @param productName (string) The name of the product that the default policy is installed as part of
+        @param filepath (string) The relative pathname to the policy file.  
+        @param repos (string) the subdirectory with the product's install directory where policy files are 
+                              stored. If None (default), the filepath argument is relative to the installation
+                              directory.
```

Comment:
This reimplements (in python) code from pex_policy. The documentation is was moved from there. You're right though, it's hard to parse. I'll clean it up.